### DISCUSSION TITLE
PDF格式的电子书转EPUB格式

### DISCUSSION THREAD

**Discussion by ailenshen** [immersive-translate/immersive-translate#1850] [Ideas]
### 新功能 | New Feature

1.通过OpenAI的接口实现对PDF格式电子书的内容提取、翻译(可选)、整理,最后转换为EPUB格式电子书

### 为什么您想要添加这些功能 | Why you want to add these features

1. EPUB格式的电子书相较PDF阅读起来更为舒适和方便,但是很多中文书籍都是PDF格式.
2. 虽然市场上有相关转换工具,但是PDF的格式直接按照固定的算法转换通常会出现很多排版异常,所以这就需要人工智能进行排版.

**Comment by theowenyoung** [comment]
https://app.immersivetranslate.com/pdf-pro/

这个 pdf-pro 就是用类似 AI 技术支持的。

**Reply by ailenshen** [reply]
PDF-PRO没有将PDF导出为EPUB功能

**Reply by theowenyoung** [reply]
> PDF-PRO没有将PDF导出为EPUB功能

html 转 epub 非常简单，您可以使用： https://convertio.co/zh/html-epub/

或者其他类似的工具： https://www.google.com/search?q=html+to+epub&oq=html+to+epub&gs_lcrp=EgZjaHJvbWUqBwgAEAAYgAQyBwgAEAAYgAQyBwgBEAAYgAQyBwgCEAAYgAQyBwgDEAAYgAQyBwgEEAAYgAQyBwgFEAAYgAQyBwgGEAAYgAQyBwgHEAAYgAQyBwgIEAAYgAQyBwgJEAAYgATSAQgyNTYxajBqMagCALACAA&sourceid=chrome&ie=UTF-8

**Reply by ailenshen** [reply]
是PDF转EPUB,而不是HTML转EPUB😊

**Reply by theowenyoung** [reply]
您可以导出 HTML 之后，再使用 html 转 epub 工具转一下。后者很简单。

**Reply by ailenshen** [reply]
好的,感谢解答.PDF-PRO看到了有导出为HTML的功能,太好了😁
